tests/test_apis.py

```diff
@@ -399,6 +399,19 @@ def testRegisterDupplicate(self):
         self.assertEqual(len(r), 2)
         self.assertEqual(list(r), ['b2', 'a'])
 
+    def testGetKeys(self):
+        r = markdown.util.Registry()
+        r.register(Item('a'), 'a', 10)
+        r.register(Item('b1'), 'b', 20)
+        r.register(Item('c'), 'c', 30)
```

Comment:
I would suggest adding one more key here, and make the calls to register out of order (to prevent this from passing by accidental sorting order).  So add maybe c, d, a, b or something.